app/controllers/api/v2/provisioning_templates_controller.rb

```diff
@@ -130,6 +131,15 @@ def export
         send_data @provisioning_template.to_erb, :type => 'text/plain', :disposition => 'attachment', :filename => @provisioning_template.filename
       end
 
+      api :GET, '/register', N_('Render Global Registration template')
+      def global_registration
+        if @provisioning_template
+          render plain: @provisioning_template.render.html_safe
+        else
+          render plain: _('Global Registration Template with name %s defined via default_global_registration_item Setting not found, please configure the existing template name first') % Setting[:default_global_registration_item], status: :not_found
```

Comment:
Just an idea, this could have been a shell comment, so when it gets executed it does not print errors.